src/containers/Footer/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { push } from 'connected-react-router'
+import { StyledFooter } from './styled';
+import { routes } from '../Router';
+import HomeOutlinedIcon from '@material-ui/icons/HomeOutlined';
+import ShoppingCartOutlinedIcon from '@material-ui/icons/ShoppingCartOutlined';
+import PersonOutlineOutlinedIcon from '@material-ui/icons/PersonOutlineOutlined';
+import BottomNavigationAction from '@material-ui/core/BottomNavigationAction';
+
+export class Footer extends Component {
+    render() {
```

Comment:
Este componente não faz uso nem do estado `this.state` quanto de nenhum métodos de lifecycle, poderia ser funcional poupando algumas linhas.